README.md

```diff
@@ -175,13 +176,20 @@ The following options are available in `build.sh`.
 |--------------------|-------------------------------------------------------------------------|
 | --snapshot         | Build a snapshot instead of a release artifact, default is `false`.     |
 | --component [name] | Rebuild a single component by name, e.g. `--component common-utils`.    |
-| --keep             | Do not delete the temporary working directory on both success or error. |
+| --keep             | Do not generate or use a stable reference manifest.                     |
+| -n, --no-lock      | Show more verbose output.                                               |
 | -v, --verbose      | Show more verbose output.                                               |
 
 ##### Custom Build Scripts
 
 Each component build relies on a `build.sh` script that is used to prepare bundle artifacts for a particular bundle version that takes two arguments: version and target architecture. By default the tool will look for a script in [scripts/components](scripts/components), then in the checked-out repository in `build/build.sh`, then default to a Gradle build implemented in [scripts/default/build.sh](scripts/default/build.sh).
 
+##### Avoiding Rebuilds
+
+Builds automatically generate `manifest.lock` files with stable git references. If no changes on any repo in the manifest have been made since the last .lock file was produced, the build process will exit successfully.
+
+You can turn off this behavior with `--no-lock`.
```

Comment:
Can we have this behavior not on by default? Seems like something that could mess up a local build if you weren't expecting it.  I would rather have the CI systems (GHA/Jenkins) opt-in

Reply:
Updated with `--lock` to enable this behavior instead of `--no-lock` to disable it.